outputs.tf

```diff
@@ -17,3 +17,8 @@ output "dl_queue_url" {
   description = "Dead letter queue url."
   value       = one(aws_sqs_queue.dlqueue[*].url)
 }
+
+output "arn" {
```

Comment:
I think `arn` as the name is too generic and could refer to anything.  I thought this was originally set to `lambda_function_arn` which would be perfect IMO